history-service/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "dotenv": "^16.3.1",
+    "express": "^4.18.2",
+    "pg": "^8.11.3"
+  }
+}
```

Comment:
also think you should fill out the other fields also like name, main, scripts, etc
https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g42/blob/master/user-service/package.json